packages/chain-adapters/src/evm/ethereum/EthereumChainAdapter.test.ts

```diff
@@ -740,7 +740,7 @@ describe('EthereumChainAdapter', () => {
       const expectedOutput = {
         txToSign: {
           addressNList: toAddressNList(adapter.getBIP44Params({ accountNumber: 0 })),
-          value: '123',
+          value: '0x7b',
```

Comment:
These two tests were obviously incorrect, the built Tx `value` property should be an hex string, not a stringified number